scripts/checkout-r2-branch.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# SUMMARY
+#   Use this script in conjuction with build-carthage-R2-integration.sh
```

Comment:
Do you mean we should run `build-carthage-R2-integration.sh` after running this script, or the other way around? The `build-carthage-R2-integration.sh` script checkout the versions we use in SE/OE for each R2 repo, so running the `build-carthage-R2-integration.sh` script after this script with the `develop` branch would not work.

Reply:
you're 100% right, I think my mistake was that I did not run the `build-carthage-R2-integration.sh` script after the "checkout" script. But yeah the intention is to run the checkout script first, then the build script. I'll fix the build script to avoid doing the checkout, since now that's done in its own script.

Reply:
fixed -- I rebased, but the only commit that changed is the first.